src/lambda_alerting.py

```diff
@@ -55,7 +56,7 @@ def write_event_to_cloudwatch(
     logged_event["event"] = event
     logged_event["monitored_environment"] = monitored_env_name
     logged_event["resource_name"] = resource_name
-    logged_event["service_name"] = service_name
+    logged_event["service_name"] = resource_type
```

Comment:
let's rename a field to "resource_type" as well

Reply:
Done